helios/templates/election_view.html

```diff
@@ -207,6 +198,55 @@ <h3 class="highlight-box">Tally</h3>
 </span> <br /><br />
 {% endif %}
 
+
+
+
+
+
+
+    </div>
+    <div class="small-12 large-3 columns" style="text-align: left;">
+        <br />
+        <div>
+            <strong>Operations:</strong>
```

Comment:
Same here, not sure the "Operations" title is helpful. I like putting all the actions in this one area, that makes sense, I might reorder them, but this is a good start. I would take out the title that is redundant though.